server/src/test/java/org/elasticsearch/action/admin/cluster/node/tasks/CancellableTasksTests.java

```diff
@@ -376,101 +372,6 @@ public void testRegisterAndExecuteChildTaskWhileParentTaskIsBeingCanceled() thro
         assertTrue("onChildTasksCompleted() is not invoked", latch.await(1, TimeUnit.SECONDS));
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/66518")
-    public void testTaskCancellationOnCoordinatingNodeLeavingTheCluster() throws Exception {
```

Comment:
Hmm this test seems somewhat useful, although I see that it's not quite valid any more. Can we make it valid again by, say, closing the connections from the "removed" nodes?

Reply:
+1, I fixed it in https://github.com/elastic/elasticsearch/pull/66525/commits/5b5712314964bc1131eb932c63353ad3fcb38167

Reply:
The adjusted test fails for me after a few iterations. Not reproducibly, otherwise I'd share a seed, but hopefully you can see this too.

Reply:
Thanks for checking. I will run it a few hundred iterations.

Reply:
I pushed the fix in https://github.com/elastic/elasticsearch/pull/66525/commits/ff8b89cb1cbd6e4d689f2f9bffacb5ce3620ffa3